openfisca_web_api_preview/loader/variables.py

```diff
@@ -38,9 +39,9 @@ def build_source_url(country_package_metadata, source_file_path, start_line_numb
 
 def build_formula(formula, country_package_metadata, source_file_path, tax_benefit_system):
     source_code, start_line_number = inspect.getsourcelines(formula)
-    if source_code[0].lstrip(' ').startswith('@'):  # remove decorator
```

Comment:
did you remove the decorator removal on purpose?

Reply:
We haven't been using these decorators for a long time (FI for this kind of off-topic cleanups I try to explicit the intention in [commit titles](https://github.com/openfisca/openfisca-core/pull/702/commits/cd36929f13fc601d1a65a98bc1c4f77b5cf7fc5c))

Agreed, it's not related to the PR.